ssllabs_exporter.go

```diff
@@ -33,10 +33,15 @@ import (
 	"github.com/anas-aso/ssllabs_exporter/pkg/ssllabs"
 )
 
+const (
+	pruneDelay = 1 * time.Minute
+)
+
 var (
-	listenAddress = kingpin.Flag("listen-address", "The address to listen on for HTTP requests.").Default(":19115").String()
-	probeTimeout  = kingpin.Flag("timeout", "Assessment timeout in seconds (including retries).").Default("600").Float64()
-	logLevel      = kingpin.Flag("log-level", "Printed logs level.").Default("debug").Enum("error", "warn", "info", "debug")
+	listenAddress  = kingpin.Flag("listen-address", "The address to listen on for HTTP requests.").Default(":19115").String()
+	probeTimeout   = kingpin.Flag("timeout", "Assessment timeout in seconds (including retries).").Default("600").Float64()
```

Comment:
the description "Assessment timeout in seconds (including retries)." is not really clear. So this timeout value is not used for a single request to SSLLab but for the whole target probe process?

Reply:
I will open another PR to cleanup some comments/logs and I will do this one on the way.